src/common/ActionButtons/ActionButtons.tsx

```diff
@@ -92,15 +92,20 @@ export const ActionButtons: FC<ActionButtonsProps> = props => {
 			openXAppOverlay={props.openXAppOverlay}
 		/>
 	));
-
+	const buttonsScreenLeaderLabel =
+		(config?.settings.customTranslations?.screenReader?.with || "With") +
+		" " +
+		buttons.length +
+		(config?.settings.customTranslations?.screenReader?.buttonsOrLinksIn ||
+			"buttons or links in");
```

Comment:
Splitting this will not make sense in several languages. I think for now, we should leave this out from customization. I will think of a different solution for this. Same goes for 'Item X of Y' in the above file